File: security/src/main/java/com/truenine/component/security/properties/JwtProperties.java
package com.truenine.component.security.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

@Data
@ConfigurationProperties(prefix = "component.security.jwt")
public class JwtProperties {
  String publicKeyClassPath = "security/pub.key";
  String privateKeyClassPath = "security/pri.key";
  String encryptDataKeyName = "edt";
  String issuer = "T-SERVER";
  Long expiredDuration = (long) (2 * 60 * 60 * 60 * 1000);
}

File: security/src/main/kotlin/com/truenine/component/security/properties/JwtProperties.kt
package com.truenine.component.security.properties

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "component.jwt")
data class JwtProperties(
  var publicKeyClassPath: String = "security/pub.key",
  var privateKeyClassPath: String = "security/pri.key",
  var issuer: String = "T-SERVER",
  var expiredDuration: Long = 2 * 60 * 60 * 60 * 1000
)